appveyor.yml

```diff
@@ -1,8 +1,20 @@
-image: Visual Studio 2019
+image: 
+  - Visual Studio 2019
+  - Ubuntu
+
 configuration: Release
     
-build:
-  project: basisu.sln
-  
+test_script:
+  - ps: |
+      New-Item -Path . -Name "build" -ItemType "directory"
+      cd build
+      cmake --version
+      cmake ../ -DCMAKE_BUILD_TYPE:STRING="Release"
```

Comment:
You can set a global `Configuration=Release` in the `appveyor.yml` file and use it here using environment variables.

See https://www.appveyor.com/docs/appveyor-yml/

Reply:
What's the environment variable?

Reply:
`$env:CONFIGURATION`?

Reply:
See the link, you need to first add a global `Configuration` section:
```
Configuration:
  - Release
```
And you can then use it using the Environment variable, using `$Env:Configuration` if I'm not wrong.

This can be used to build multiple configurations. It's just a suggestion.

Reply:
Do you know if `Configuration` can be used in artifacts somehow?

Reply:
I think you can